sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -24,6 +25,7 @@ public static final class RuleViolationTemplates {
         // Format: First %s is either "abstract class" or "interface".
         //         Second %s is the name of a class or interface.
         public static final String UNIMPLEMENTED_TYPE_RULE = "Extend, implement, or delete %s %s";
+        public static final String UNUSED_METHOD_RULE = "Method %s in class %s is never invoked";
```

Comment:
@teresa-allen-sfdc , same question here: Do we want to change the violation message to mention paths?

Reply:
Since we have the concept of external sources, can we say, "Method %s in class %s is never invoked from any of the recognized sources" or something to indicate we have a notion of entry-point/source?